blog/2024/10-26-media-segments-10.10.0/index.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Jellyfin Media Segments Explained 10.10.0
```

Comment:
```suggestion
title: Media segments explained
```

No need to mention the product name or version in the title, you're already on our blog where all that information is readily available